sanity-io/structured-content-2022
# Newsletter section fixes

# Newsletter section fixes

## Intent

- Replace placeholder image for _ConferenceUpdatesForm.tsx_ `'newsletter'`-type section 
- Deletes unused PlaceholderImage component
- Creates [/confirmation](https://structured-content-2022-web-pp63ilshf.sanity.build/confirmation) page in Studio

## Description

The [/confirmation](https://structured-content-2022-web-pp63ilshf.sanity.build/confirmation) page is added to match up with the current redirect that occurs when a user submits a previously non-subscribed email address into the `'newsletter'`-type form. Otherwise, this redirect would lead to a 404 after we merge `staging` to `master`. 

I looked throughout Mailchimp, but could not find the place where this redirect is configured. No matter; I like this solution because it means I don't have to prod any Sanity people to find and update this redirect. That also means the redirect won't properly work prior to launch, but it can still be accessed by visiting the path.

## Testing this PR

1. Verify that the [frontpage](https://structured-content-2022-web-pp63ilshf.sanity.build/) (and other pages) render the "final" image for the "Get conference updates" form in large viewports
2. Submit a previously non-subscribed email in the "Get conference updates" form and verify that you're redirected to /confirmation 
3. Verify that this matches the slug of the [Page](https://stagingadmin.structuredcontent.live/desk/page;2ef28313-b975-407f-9953-19a98a45cf84) and [Route](https://stagingadmin.structuredcontent.live/desk/route;8ea62f43-6009-4cc2-931f-09841af721f8) I created in Studio
4. Visit /confirmation and verify that the page renders without any console warnings or errors
